tests/ui_tests/utils.py

```diff
@@ -167,6 +168,21 @@ def check_rows_amount(page, expected_amount: int, table_page_num: int):
     ) == expected_amount, f'Page #{table_page_num} should contain {expected_amount}, not {row_count}'
 
 
+def ignore_flaky_errors(func: Callable):
+    """
+    Use it as decorator to catch flaky exceptions and raise Assertion messages instead.
+    Use it only with `wait_until_step_succeeds` or something similar.
+    """
+
+    def wrapped(*args, **kwargs):
+        try:
+            func(*args, **kwargs)
+        except (StaleElementReferenceException, NoSuchElementException) as e:
+            raise AssertionError('Got a flaky error: %s' % e) from e
```

Comment:
I guess you should use f-string here.

Reply:
yep, % is preferred for error message formatting, but here it serves no purpose. Changed